packages/mdc-chips/chip/mdc-chip.scss

```diff
@@ -36,6 +39,7 @@
   cursor: pointer;
   outline: none;
   overflow: hidden;
+  vertical-align: middle;
```

Comment:
This is necessary even though we're inline-flex and already have align-items: center?

Reply:
You're right, removed.